PIconnect/PIData.py

```diff
@@ -187,6 +195,31 @@ def current_value(self):
         Return the current value of the attribute."""
         return self._current_value()
 
+    def update_value(
+        self,
+        value,
+        ddatetime: Optional[datetime] = None,
```

Comment:
I think `timestamp` would be a better name for this parameter without conflicting with the `datetime` class name.

Reply:
this should then also be reflected in the tutorial.

Reply:
ok, what I have thought about when I tried to choose b/w:
1. time - this name represents only time, i.e. without date info, thus as a name for instance that shows time and date simultaneously it is not suited well.
2. ddatetime - it came as a kinda compromise b/w 1 point and already existed datetime type
3. timespan - it is not suited even more than in the 1 point

What should we do?
I guess, let's stop on simple "time":
- you use it in methods: recorded_values(..., start_time, ...), interpolated_values(..., start_time, ...). So, we will follow module`s style
- the 1 point mentioned above is questioned more than 3d, so... 
- ddatetime looks like a pitfall, dt is a more correct one. But, follow module style is better